src/project/internal/exportprojectscenario.h

```diff
@@ -19,25 +19,25 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-#ifndef MU_USERSCORES_EXPORTSCORESERVICE_H
-#define MU_USERSCORES_EXPORTSCORESERVICE_H
+#ifndef MU_PROJECT_EXPORTPROJECTSCENARIO_H
+#define MU_PROJECT_EXPORTPROJECTSCENARIO_H
 
 #include "modularity/ioc.h"
 
-#include "iexportscorescenario.h"
-#include "project/iprojectconfiguration.h"
+#include "iexportprojectscenario.h"
+#include "iprojectconfiguration.h"
 #include "iinteractive.h"
-#include "project/inotationwritersregister.h"
+#include "inotationwritersregister.h"
 #include "importexport/imagesexport/iimagesexportconfiguration.h"
 #include "context/iglobalcontext.h"
 #include "system/ifilesystem.h"
 
-namespace mu::userscores {
-class ExportScoreScenario : public IExportScoreScenario
+namespace mu::project {
+class ExportProjectScenario : public IExportProjectScenario
 {
-    INJECT(userscores, project::IProjectConfiguration, configuration)
+    INJECT(userscores, IProjectConfiguration, configuration)
```

Comment:
please rename userscores to project here

Reply:
Thanks, fixed